codetransformer/code.py

```diff
@@ -551,13 +553,37 @@ def firstlineno(self):
 
     @property
     def lnotab(self):
-        """The encoding of address to lineno mapping.
-
-        See Objects/lnotab_notes.txt for details.
+        """The mapping of instructions to the line that they start.
         """
         return self._lnotab
 
     @property
+    def py_lnotab(self):
+        """The encoded lnotab that python uses to compute when lines start.
+
+        Note
+        ----
+        See Objects/lnotab_notes.txt in the cpython source for more details.
+        """
+        lnotab = self.lnotab
+        py_lnotab = bytearray(len(lnotab) * 2)
+        idx = 0
+        last_instr = 0
```

Comment:
Can we call this `previous` rather than last?  last is ambiguous with the actual last instruction in the code object .